VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioComponent.cpp

```diff
@@ -957,30 +964,44 @@ TArray<UGeneratedModelHISMComponent*> UVitruvioComponent::GetGeneratedModelHISMC
 	return VitruvioModelHISMComponents;
 }
 
+FString UVitruvioComponent::GetMaterialIdentifier(const UMaterialInterface* SourceMaterial) const
+{
+	if (const FString* Result = MaterialIdentifiers.Find(SourceMaterial))
+	{
+		return *Result;
+	}
+
+	return {};
```

Comment:
Maybe we should add an ensure here to check that this code is never reached.

Reply:
Good point. Will add an ensure and change the doc.